src/adminhandler.hpp

```diff
@@ -150,13 +150,15 @@ class AdminHandler : public osmium::handler::Handler
                                        parent_admin_levels.end()) !=
                     parent_admin_levels.end();
 
+               const std::string linestring = m_factory.create_linestring(way);
+
```

Comment:
I'm viewing this on my phone so I'm not 100% sure, but the indentation looks different here.

Reply:
Oh, wow, yes it is. Good catch. I've fixed it up and added a comment to help explain why this is not in the `<<` lines.